test/testAdBlock.js

```diff
@@ -6,6 +6,7 @@ import path from 'path'
 import { spawnSync } from 'child_process'
 
 import { generateResourcesFile, sanityCheckList } from '../lib/adBlockRustUtils.js'
+import { preprocess } from '../scripts/generateAdBlockRustDataFiles.js'
```

Comment:
could we move `preprocess` to `adBlockRustUtils`?

Reply:
done